.github/workflows/unlock-reopened-issue-pr-with-message.yml

```diff
@@ -0,0 +1,40 @@
+name: Unlock Reopened Issues and PRs with message
+
+on:
+  pull_request_target:
+    types: [reopened]
+  issues:
+    types: [reopened]
+
+jobs:
+  auto_comment:
+    permissions:
+      pull-requests: write
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aws-actions/closed-issue-message@v2
```

Comment:
@rix0rrr how do you feel about reusing this here? Despite the name, the action isn't actually specific to closed issues: https://github.com/aws-actions/closed-issue-message